### PR TITLE
doc updates

### PR DESCRIPTION
**Pull request by jamie-short**
Few doc updates relating mainly to the login and logout commands

### CODE REVIEW

File: docs/dafni_cli.md [hunk 1]
```diff
@@ -2,24 +2,43 @@
...
 
+The current method of distribution is via an executable program that can be run on the command line. This will take the form of a executable file called `dafni.exe`.
+Due to this, there is no installation at this stage from a user perspective, just a necessity to have a copy of the executable on the computer it is to be used on.
+___
+
 ## Commands
 
-The syntax for these will change as per the requirements i.e.\
+The syntax for these will change as per the requirements i.e.  
 `dafni get model` instead of `dafni model get`.
```
Comment (iroper-tessella) [inline docs/dafni_cli.md:12]: Do you mean that the ".exe" will eventually be dropped?

**Review by iroper-tessella — COMMENTED** [review]
Just one small question on the syntax

File: docs/dafni_cli.md (hunk 1 above)
Comment (jamie-short) [reply docs/dafni_cli.md:12]: So that was already there actually, although reading through it can likely be dropped entirely

Comment (iroper-tessella) [reply docs/dafni_cli.md:12]: Oh, that's probably from me then. Yeah, we can drop this.

**Review by iroper-tessella — APPROVED** [review]
Yep, happy to merge now.

File: docs/dafni_cli.md (hunk 1 above)
Comment (jamie-short) [reply docs/dafni_cli.md]: removed

**Pull request merged**